scio-smb/src/test/avro/user.avsc

```diff
@@ -0,0 +1,10 @@
+{
+  "namespace": "org.apache.beam.sdk.extensions.avro.io",
+  "type": "record",
+  "name": "AvroGeneratedUser",
```

Comment:
did the Beam Avro class get deleted?

Reply:
The artifact is published with a class compiled with another, incompatible avro version